### PR TITLE
Update readme.md and fix linter

### PR DESCRIPTION
**Pull request by leviadam**
This PR updates the npm package on readme.md.
It also fixes the linter by updating the package-lock file and removing an unused variable in templates.js.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,24 +34,24 @@ npm install -g truffle
 mkdir myproject && cd myproject
 truffle init
 ```
-To install the daostack-arc library, run:
+To install the @daostack/arc library, run:
```
Comment (orenyodfat) [inline README.md]: Should it be : 

> To instal arc package ,run : 
```sh
 npm init
 npm install @daostack/arc
  ```		  ```

Comment (leviadam) [reply README.md]: You can do `npm install` without `npm init`.

**Comment by orenyodfat** [comment]
Yea, my comment is re "To instal arc package ,run :"
Replace library with package

**Comment by leviadam** [comment]
Agree.

**Pull request merged**